src/core/server/services/perspective/perspective.ts

```diff
@@ -169,6 +176,10 @@ export async function sendToPerspective(
       );
     }
 
+    if (err.message && err.message.startsWith("axios request timed out")) {
+      logger.error("perspective request timed out");
```

Comment:
Instead of logging here, throw an error that matches the same error text as the fetch API's error message when the timeout occurs, then @immber doesn't need to update the existing reporting flows to track incidents.